etc/poco/bundle/main-test.json

```diff
@@ -87,6 +87,14 @@
 			"mechanism": "asap",
 			"principals": ["micros/manager-service"],
 			"allowed": true
+		},
+		{
+			"name": "Allow support team to call API endpoints",
+			"path": "/api/hars",
+			"method": "GET",
+			"mechanism": "open",
+			"principals": [],
```

Comment:
You should add support team here as the only allowed principal, and then add another test where allowed: false for a random test principal